src/mmw/js/src/core/views.js

```diff
@@ -2,7 +2,7 @@
 
 var $ = require('jquery'),
     L = require('leaflet'),
-    _ = require('lodash'),
+    _ = require('underscore'),
```

Comment:
Might be related to: https://github.com/WikiWatershed/model-my-watershed/issues/450